css/style.css

```diff
@@ -73,6 +73,68 @@ aside {
   padding: 5px;
 }
 
+#navOptions #story-li.active {
+  background-color: #aacfe3;
+  padding-top: 25;
```

Comment:
a bit redundant definition of padding and transition. u could separate that out from the color definition